### PR TITLE
Rescript/react

### PR DESCRIPTION
**Pull request by JanKaul**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,8 @@
 .DS_Store
 *.log
 node_modules
+package-lock.json
+yarn.lock
```
Comment (MoOx) [inline .gitignore]: On or the other is mandatory :)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,12 @@
 # Changelog of `@moox/bs-react-helmet`
 
+## 4.0.0 - 2021-09-23
+
+- removed bs-platform and reason-react
+- added rescript and @rescript/react
+
+Bindings to helmetProp changed.
+
```
Comment (MoOx) [inline CHANGELOG.md]:
```suggestion
```

The changelog should be edited after so should be remove.

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@moox/bs-react-helmet",
-  "version": "3.0.0",
+  "version": "4.0.0",
```
Comment (MoOx) [inline package.json]:
```suggestion
  "version": "3.0.0",
```

Should be changed on release.

**Review by MoOx — COMMENTED** [review]
Btw, should be renamed but I can do that after and deprecate the old package.

File: .gitignore (hunk 1 above)
Comment (JanKaul) [reply .gitignore]: I'm actually quite new to nodejs and npm, so I didn't know its mandatory. If you prefer yarn, we can also keep yarn.lock.

**Comment by JanKaul** [comment]
Thank you so much for your comments!

File: .gitignore (hunk 1 above)
Comment (MoOx) [inline .gitignore]:
```suggestion
```

Comment (MoOx) [reply .gitignore]: I would prefer to keep the yarn lock file and do not ignore any of those files.

**Review by MoOx — APPROVED** [review]

**Comment by MoOx** [comment]
Sorry for the delay, I didn't use this module for a while.

**Pull request merged**